acceptance/run.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-set -e
+set -eu
+if [ -n "${BASH:-}" ]; then
```

Comment:
This file (and many of the others you've modified) have `#!/bin/bash`, so you can use `set -o pipefail` unguarded. In general, though, I would avoid deviations from the standard shell environment. We shouldn't have 5-line preambles for all of our shell scripts. I think that `set -e` is a worthwhile habit, but the other settings are questionable (for example, this file looks unsafe for use with `set -u` when run directly instead of through the makefile). Only set pipefail and IFS when it matters.